docs/native-modules-ios.md

```diff
@@ -73,7 +73,7 @@ This argument is not a string literal. In the example below `RCT_EXPORT_MODULE(C
 RCT_EXPORT_MODULE(CalendarModuleFoo);
 ```
 
-The native module can then be accessed in JS like this:
+Once we finish building our native module, we will be able to access it in JS like this:
```

Comment:
Please revert this wording as we never mentioned "building" before (in this and other changes).

Reply:
@cortinico hi. I have reverted this specific change from both the Android and iOS pages.